pkg/habitat_client/client.go

```diff
@@ -112,6 +116,72 @@ func PostRequestToAddress(address string, req, res interface{}) (error, error) {
 	return nil, nil
 }
 
+func PostFileToAddress(address string, client *http.Client, file *os.File, res interface{}) (error, error) {
```

Comment:
hmm wondering if these belong in `habitat_client/client.go` or just in `fs.go` since thats the only call point

Reply:
I think its generic enough that it could be reused in the future